app/grandchallenge/challenges/models.py

```diff
@@ -957,9 +957,18 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
         if adding:
+            self.assign_permissions()
             send_challenge_requested_email_to_reviewers(self)
             send_challenge_requested_email_to_requester(self)
 
+    def assign_permissions(self):
+        assign_perm("view_challengerequest", self.creator, self)
+        reviewers, _ = Group.objects.get_or_create(
```

Comment:
I think only get the reviewers here. You can create the group in a post migration step in `apps.py`, see the implementation of the registered users group.